### ISSUE TITLE
Heading "weapon descriptions" has the description of a single weapon in it

### ISSUE THREAD

**Issue by Daniel-Soham** [mlenser/kryx-rpg-issues#998]
**Describe the bug**
Weapon descriptions is given a heading and then a single weapon is there. Why? Either more weapons should have descriptions here for more unusual weapons, or all weapons should have a description, or the section should be deleted.

Weapon descriptions
Sling Staff. This staff ends in a Y-shaped split that cradles a sling. The length of the staff provides excellent leverage when used two‑handed to fling rocks or bullets from the sling.

**Link**
https://www.kryxrpg.com/equipment/weapons

**Comment by mlenser** [comment]
This is correct - only one weapon (the sling staff) has extra text associated with how it is shaped.

All weapons could probably use some description, but that doesn't exist in the original source material.

**Comment by Daniel-Soham** [comment]
Well the section is technically one weapon description then and not weapon
descriptions then. A bit odd.
